.changeset/shy-socks-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ai-sdk/core': patch
```

Comment:
This should be a minor release in my opinion.

Reply:
I would say from the users perspective, the impacted packages are orchestration and foundation-models.

Reply:
I changed it to minor.

Regarding the second point, I am still in favour of noting the changes as part of the `core` package. As almost all functionality in `core` is meant to be used by other packages, it will basically ask us to never put `core` in release note. And some changes could affect very widely, such as introducing `ErrorWithCause` in very low level in `core`. In such case, I prefer not to put all other packages here except `core`, even if all other packages has zero code changes.

Reply:
Sorry for the confusion, I didn't imply removing core from affected packages in the changeset, rather also including the other 2.
Since we say core is internal and not for users, the changelog might imply that the change is restricted to core. However, the main change is seen in the orchestration/foundation-models by the users. The release notes are mainly for public api changes.